include/battle/battle.h

```diff
@@ -623,11 +626,8 @@ struct BattleSystem {
     u16 unk2454[4];
     u16 unk245C[4];
     int unk2464[4];
-    u32 isRecordingPaused : 1,
-        unk2474_1         : 1,
-        unk2474_2         : 1,
-        unk2474_3         : 1,
-        unk2474_4         : 28;
+    u32 isRecordingPaused : 1, unk2474_1 : 1, unk2474_2 : 1, unk2474_3 : 1,
```

Comment:
hey clang-format, you okay?

Reply:
clearly not but we should probably put a u32 in front of all of these anyways